pattoo/api/portal/react/src/Datapoint.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import DatapointHead from './DatapointHead';
```

Comment:
Where are the comments on what you are doing and why? This is important for future developers.